cmd/kubernetes/Dockerfile

```diff
@@ -1,23 +1,23 @@
+# Build UI
```

Comment:
This Kubernetes Dockerfile isn't depended on the UI. 
Isn't it possible to "just" build the Go binary?

Don't worry about it tooo much though, as I'm planning to just have a single `pyrra` binary with these as sub commands.